pages/facts.html

```diff
@@ -4,9 +4,73 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Facts</title>
+  <link href="../styles/style.css" rel="stylesheet" />
+  <link href="../styles/facts.css" rel="stylesheet" />
+
 </head>
 <body>
-  
+  <header>
+    <h1>All about Pizza</h1>
+    <h2>Group 11 Fansite</h2>
+    <nav>
+      <ul>
+        <li><a href="../index.html">Home</a></li>
+        <li><a href="gallery.html">Gallery</a></li>
+        <li><a href="facts.html">Fun Facts</a></li>
+        <li><a href="faq.html">Frequently Asked Questions</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+    <section class="grid-container">
+      <div class="fun-facts">
+        <h3>Fun Facts 1: World’s Largest Pizza</h3>
+        <p>The world’s largest pizza was made in Rome in 2012 and measured over 13,500 square feet! 
+          It was gluten-free and named “Ottavia” in honor of the first Roman emperor, Octavian Augustus.
+        </p>
+      </div>
+
+      <div class="fun-facts">
+        <h3>Fun Facts 2:Americans Love Their Pizza</h3>
+        <p>
+          Each year, Americans eat about 3 billion pizzas, which averages to 46 slices per person! Pepperoni is the most popular topping.
+        </p>
+      </div>
+
+      <div class="fun-facts">
+        <h3>Fun Facts 3: The First Pizzeria</h3>
+        <p>
+          The world’s first pizzeria, Antica Pizzeria Port'Alba, opened in Naples, Italy, in 1738. It’s still in business today!
+        </p>       
+      </div>
+
+      <div class="fun-facts">
+        <h3>Fun Fact 4: National Pizza Month</h3>
+        <p>
+          October is officially National Pizza Month in the United States. It was created in 1984 by a pizza industry publisher to celebrate one of America’s favorite foods!
+        </p>       
+      </div>
+
+      <div class="fun-facts">
+        <h3>Fun Fact 5: A Pizza in Space</h3>
+        <p>
+          In 2001, Pizza Hut became the first company to deliver pizza to space. A pizza was sent to the International Space Station for a Russian cosmonaut!
+        </p>
+      </div>
+
+      <div class="fun-facts">
+        <h3>Fun Fact 6: The Most Expensive Pizza</h3>
+        <p>
+          The world’s most expensive pizza costs a whopping $12,000! The Louis XIII Pizza is topped with luxury ingredients like caviar, lobster, buffalo mozzarella, and 24-karat gold flakes.
+        </p>        
+      </div>
+    </section>
+  </main>
+
+  <footer>
+    &copy; Group 11, Cohort 23
+  </footer>
```

Comment:
To keep a consistent look and feel across a website's pages, we typically want to keep heading/navigation and footer content in sync across all pages.